doc/api/errors.md

```diff
@@ -128,12 +128,12 @@ they are thrown *after* the calling code has already exited.
 Developers must refer to the documentation for each method to determine
 exactly how errors raised by those methods are propagated.
 
-### Node.js style callbacks
+### Error-first callbacks
 
 <!--type=misc-->
 
 Most asynchronous methods exposed by the Node.js core API follow an idiomatic
-pattern  referred to as a "Node.js style callback". With this pattern, a
+pattern  referred to as an _error-first callback_. With this pattern, a
```

Comment:
For historical reasons it would be good to keep a reference to the Node.js style callback as in:
```
... error-first callback, historically called `Node.js style callback`.
```
The reason is that e.g. blog posts and other documentations might still call them like this and it would be good to find a explanation even with the new name.

Reply:
Rather than `historically called` I'd prefer the simpler and arguably-more-accurate `sometimes called`...

`... a_error-first callback_ (sometimes referred to as a _Node.js style callback_) ...`

Reply:
sounds correct to me as well. Done 👍